src/identity/selectors/index.ts

```diff
@@ -6,6 +6,12 @@ export const selectQuartzIdentity = (state: AppState): AppState['identity'] => {
   return state.identity
 }
 
+export const selectCurrentIdentity = (
+  state: AppState
+): AppState['identity']['currentIdentity'] => {
+  return state.identity.currentIdentity
+}
+
```

Comment:
The identity hierarchy in state contains two objects - `currentIdentity` (the current user, account, and org) and `quartzOrganizations` (the array of all orgs in the current account).

We want a more specific selector for other parts of the app that just refers to `currentIdentity` to ensure we're keeping re-renders to a minimum.